### PR TITLE
finished gun catalogue (SJE-04)

### PR DESCRIPTION
**Pull request by Sennarin**
image

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -17,6 +17,159 @@
 <!-- game modes finish -->
 
 <!-- guns catalogue start -->
+<section class="guns" id="guns">
```
Comment (mosiichuk) [inline src/index.html:20]: Please think regarding naming, cause for now, you have section guns, and then `guns__item`, `guns__title`. But the problem is that we can't use BEM in such way, cause guns section is not reusable block. For example `guns__title` should be section header or something like that, it's reusable and independent block. Also, here we see repeatable guns images, that possible could be used in other part of the site. We should have one class `nerf-gun-thumbnail` for small images, and `nerf-gun-lg` for large, only size will be in css, no borders or other stuff should be added here. And then we should create classes for catalog. 
Should be something like this: 
1) `catalog`
image
2) `catalog__row`
image
3) `catalog-item` will be the block.
3.1 `catalog-item-preview` this class will give the element action on hover and border. (basically will replace classes `guns__bg` and `guns__item`)
image
3.2 `catalog-item__img`, `catalog-item__info` `catalog-item__title`, `catalog-item__desc` 
image

File: src/styles/style.sass [hunk 2]
```diff
@@ -68,6 +69,71 @@ a
...
+
+    &__desc
+        padding: 15px 0px
+        margin-top: 20px
+        width: 290px
+        height: auto
+        background: $accent-orange
+
+    &__text
+        margin-top: 10px
```
Comment (mosiichuk) [inline src/styles/style.sass:134]: Don't use margin top. Instead margin-bottom for the above element or padding for the parent.

File: src/styles/style.sass [hunk 3]
```diff
@@ -68,6 +69,71 @@ a
...
+            transform: scale(2)
+            transition: 0.5s all ease
+
+        &:hover .guns__bg
+            border: 3px solid $accent-blue
+            transition: 0.5s all ease
+
+    &__list
+        margin-bottom: 64px
+        &:last-child
```
Comment (mosiichuk) [inline src/styles/style.sass:101]: Add new line above the `&:last-child`

File: src/styles/style.sass [hunk 4]
```diff
@@ -68,6 +69,71 @@ a
 // <!-- game modes finish -->
 
 // <!-- guns catalogue start -->
+
+.guns
+    padding: 60px 0px
+
+    &__block
```
Comment (mosiichuk) [inline src/styles/style.sass:76]: Let's avoid word block in class namings, as we use BEM, where B is meant for block.

File: src/styles/style.sass [hunk 5]
```diff
@@ -68,6 +69,71 @@ a
...
+        margin-top: 35px
+
+    &__title
+        width: 360px
+        height: 64px
+        background: $accent-orange
+
+    &__item
+
+
```
Comment (mosiichuk) [inline src/styles/style.sass:86]: Remove extra new line after `&__item`. Should be 1 instead of 2.

File: src/styles/style.sass [hunk 6]
```diff
@@ -68,6 +69,71 @@ a
...
+            width: 80px
+            height: 80px
+
+.big-gun
+    margin-top: 15px
+    background: url('../img/gun-bg.png') center top / cover
+    width: 289px
+    height: 289px
+
+    &-wrapper
```
Comment (mosiichuk) [inline src/styles/style.sass:124]: Let's avoid empty classes in css.

File: src/index.html (hunk 1 above)
Comment (Sennarin) [reply src/index.html:20]: jesus, this one took a while to fix

**Pull request merged**